### ISSUE TITLE
input-debounce not working on q-select

### ISSUE THREAD

**Issue by aryan-myrtle** [quasarframework/quasar#16377]
### What happened?

input-debounce does not work on q-select when q-select used as input -- combobox.

### What did you expect to happen?

input-debounce="1000" should delay the execution of @input-value by 1 second.

### Reproduction URL

https://jsfiddle.net/kgbhr7s9/5/

### How to reproduce?

1. open jsfiddle
2. type in the input
3. it should debounce the console.log function
4. but nothing is being debounced

### Flavour

Vite Plugin (@quasar/vite-plugin)

### Areas

Components (quasar)

### Platforms/Browsers

Chrome

**Comment by HarisSpahija** [comment]
Hi Aryan,

In your example you made a couple of implementation mistakes. 
To use the delay you have to make sure you pass a number.

Also the delay is used for delaying a filter function.

For example using your component 

```vue
 <q-select
    v-model="departureCity"
    use-input
    :input-debounce="1000"
   @filter="justdoit"
>
</q-select>
```

Input will always happen. Perhaps the prop name could be a bit better. Something like `filter-debounce` would make more sense I believe. Hope this helps!

**Comment by rstoenescu** [comment]
Improvement/fix (see commit msg above) will be available in Quasar v2.12.8
